Makefile

```diff
@@ -17,6 +17,8 @@ MIGRATION_PLANNER_API_IMAGE_PULL_POLICY ?= Always
 MIGRATION_PLANNER_NAMESPACE ?= assisted-migration
 MIGRATION_PLANNER_REPLICAS ?= 1
 MIGRATION_PLANNER_AUTH ?= local
+MIGRATION_PLANNER_ISO_URL ?= https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/rhcos-live-iso.x86_64.iso
```

Comment:
So let's stick to specific version? Because the image can change and SHA below won't change, so in future this may not work?

Reply:
You mean to use explicitly rhcos-4.19.0-x86_64-live-iso.x86_64.iso instead of the alias?
Well yes, if the computed sha does not match the one in the config, the download fails yes.

Reply:
So each time they bump the version, our app stops to work with current approach, if I am not mistaken.